2015-Fall/soerendalby/Calculater/ViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ViewController.swift
+//  Calculater
+//
+//  Created by Søren Dalby on 06/09/15.
+//  Copyright (c) 2015 Søren Dalby. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+ 
+    @IBOutlet weak var display: UILabel!
+
+    var continueCurrentNumber: Bool = false
```

Comment:
Here there is no need to specify the actual type of the variable. It is inferred to boolean and clear from then context. `var continueCurrentNumber = false`